app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str):
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill):
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self, name: str):
+        super().__init__(name)
+        for skill in ["JavaScript", "HTML", "CSS"]:
```

Comment:
Here and below, maybe better use "extend"  instead of "for", but it's up to you.